examples/src/main/python/mllib/streaming_linear_regression_example.py

```diff
@@ -25,13 +25,14 @@
 # $example off$
 
 from pyspark import SparkContext
-from pyspark.streaming import StreamingContext
 # $example on$
 from pyspark.mllib.linalg import Vectors
-from pyspark.mllib.regression import LabeledPoint
-from pyspark.mllib.regression import StreamingLinearRegressionWithSGD
+from pyspark.mllib.regression import (LabeledPoint,
+                                      StreamingLinearRegressionWithSGD)
```

Comment:
I actually prefer having a single import per line (this simplifies a lot file management, multi branch merges,...). I can revert this change